sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/DotNetSensorTest.java

```diff
@@ -210,30 +210,33 @@ public void whenThereAreOnlyMainFilesInAnotherLanguage_logOnlySkipSensor() {
   }
 
   @Test
-  public void whenThereAreOnlyTestFilesInPluginLanguage_andNoMainFilesInAnyLanguage_logConsoleAndAnalysisWarnings() {
+  public void whenThereAreOnlyTestFilesInPluginLanguage_andNoMainFilesInAnyLanguage_resultsAreImportedWithoutWarnings() {
```

Comment:
this should still warn, as mentioned [here](https://github.com/SonarSource/sonar-dotnet/pull/4208#discussion_r607678662)